build.gradle.kts

```diff
@@ -327,7 +327,7 @@ tasks.register("python39PostCommit") {
   dependsOn(":sdks:python:test-suites:direct:py39:postCommitIT")
   dependsOn(":sdks:python:test-suites:direct:py39:hdfsIntegrationTest")
   dependsOn(":sdks:python:test-suites:portable:py39:postCommitPy39")
-  dependsOn(":sdks:python:test-suites:dataflow:py39:torchTests")
+  dependsOn(":sdks:python:test-suites:direct:py39:inferencePostCommitIT")
```

Comment:
Do we also want to add `dependsOn(":sdks:python:test-suites:direct:py38:inferencePostCommitIT")`